autoPyTorch/ensemble/ensemble_builder.py

```diff
@@ -274,7 +274,7 @@ def fit_and_return_ensemble(
     ensemble_nbest: int,
     max_models_on_disc: Union[float, int],
     seed: int,
-    precision: int,
+    precision: Union[int, str],
```

Comment:
I think we should only use int. It happened in autosklearn that we were comparing `'128' == 128`  which is false, and had unexpected behavior.

Reply:
I agree with you.
Preferably, we would like to use `typing.Literal`, but it is not supported python<=3.7, so typing like `np.int64` also works better.
Anyways, we only have `32` or `64`, right? Then it is better to define by constant variables.

Reply:
okay I'll go with int then